Firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [fixed] Fixed a crash that would happen when the app is being deleted and
```

Comment:
Is there a way to simplify this phrasing? Both "the app is being deleted" and "immediately disposed of" sound duplicative. Also, should we capitalize "app"?

Reply:
Sorry, I got caught up in CI failures and somehow missed this comment. Addressed in #7449.